### PR TITLE
Fix scroll bugs with vertical scrollbar

### PR DESCRIPTION
**Pull request by yotamberk**

### CODE REVIEW

File: lib/timeline/Core.js [hunk 1]
```diff
@@ -202,29 +207,29 @@ Core.prototype._create = function (container) {
...
   } else {
     // IE 6/7/8
-    this.dom.root.attachEvent("onmousewheel", onMouseWheel.bind(this));
+    this.dom.center.attachEvent("onmousewheel", onMouseWheel.bind(this));
   }
 
   function onMouseScrollSide(event) {
-    var current = this.scrollTop;
-    var adjusted = -current;
+    if (!me.options.verticalScroll) return;
```
Comment (Tooa) [inline lib/timeline/Core.js]: You should add curly braces here in order to be consistent with the code style.

File: lib/timeline/Core.js [hunk 2]
```diff
@@ -1070,6 +1072,7 @@ Core.prototype._onPinch = function (event) {
  * @private
  */
 Core.prototype._onDrag = function (event) {
+  if (!event) return
```
Comment (Tooa) [inline lib/timeline/Core.js]: Add curly braces and missing `;`. See my comment above.

File: lib/timeline/Timeline.js [hunk 3]
```diff
@@ -109,6 +104,11 @@ function Timeline (container, items, groups, options) {
   // current time bar
   this.currentTime = new CurrentTime(this.body);
   this.components.push(this.currentTime);
+  
+  // apply options
+  if (options) {
```
Comment (Tooa) [inline lib/timeline/Timeline.js]: Here we have the braces. Is this some Javascript style convention to omit them if it's a return? I'm not sure.

File: lib/timeline/Range.js [hunk 4]
```diff
@@ -390,6 +390,8 @@ Range.prototype._onDragStart = function(event) {
  * @private
  */
 Range.prototype._onDrag = function (event) {
+  if (!event) return
```
Comment (Tooa) [inline lib/timeline/Range.js]: See above.

File: lib/timeline/Core.js (hunk 2 above)
Comment (mojoaxel) [inline lib/timeline/Core.js]: I thinks, you can leave it as it is.

**Comment by yotamberk** [comment]
@mojoaxel I kind of prefer without the curly braces. But it is a question of style. I think you can accept it like this. If we decide later differently, i'll be glad to change it.

**Pull request merged**